src/NUnitFramework/framework/Attributes/ApartmentAttribute.cs

```diff
@@ -29,7 +29,7 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// Marks a test that must run in a particular threading apartment state, causing it
+    /// <see cref="ApartmentAttribute"/> marks a test that must run in a particular threading apartment state, causing it
```

Comment:
While I would use crefs any time a code element is named, I wouldn't name the attribute here. [The BCL doesn't do this](https://github.com/dotnet/corefx/blob/d4541b21dcb0cb15c5ec49c38354bde46e3eaea4/src/System.ComponentModel.Primitives/src/System/ComponentModel/DescriptionAttribute.cs#L7-L11) and the reason that occurs to me is that the name is already right above the text, so starting off the sentence with the same name is visually redundant:

![image](https://user-images.githubusercontent.com/8040367/44031421-fadf55f2-9ed1-11e8-9d67-6a6d2b218c1a.png)

I know it's confusing that I didn't ask to entirely remove the existing content in https://github.com/nunit/nunit/pull/2952, but it did cross my mind. :D I love that you're shooting for consistency. That's something we could really use. If you're willing to help us in this direction, I'd prefer to start the sentence with the class name being an implicit subject noun like I see in other projects.

Reply:
Checking with @nunit/framework-team, is view you'd share?

Reply:
@jnm2 not a problem.  Lets get it sorted.  As I said to Charlie, I'm willing to redo them so they are all have the same structure and consistency.  It isn't a problem.  I'll wait until the general consensus is back and then we can go from there.

Reply:
> Checking with @nunit/framework-team, is this a view you'd share?

I'm apathetic. Happy to accept this, as @sboptimal's put the work in to tidy things up, but I have no strong opinion on whether we're consistent one way, or the other. 🙂

Reply:
Okay :D Just to clarify, there's a ton in this PR that I want to merge whichever way we go. I'm proposing that in addition to these changes (and as a reversal of only some of them) we delete the attribute names as the first words of the summaries and start with the next word.

Reply:
Sounds good to me!

Reply:
I don't feel strongly about this. If @sboptimal doesn't minding doing the additional changes then I don't see any reason not to do this. Otherwise, I can also live with the current solution.

Reply:
Ok, so just to confirm:
1. The Cref at the start of each summary will be removed along with the attribute name
1. All attributes will now start with something like: Marks..., Applies..., Supplies... etc.  Basically, whatever their first word is after the attribute name.

Old:
```
    /// <summary>
    /// <see cref="RandomAttribute"/> supplies a set of random values
    /// to a single parameter of a parameterized test.
    /// </summary>
```

New:
```
    /// <summary>
    /// Supplies a set of random values to a single parameter of a parameterized test.
    /// </summary>
```

Looking at the BCL link you sent through earlier.  It seems they wrap the summary comments in a <para> tag for formatting reasons I guess.  Do you want your attributes doing the same way as i'll do them at the same time if so?

Reply:
@sboptimal That's what I had in mind, yes. We don't want to use `<para>` tags line the BCL did in that example, though. We're targeting only IntelliSense with our XML docs and we only use `<para>` to wrap at least two paragraphs.

Reply:
Ok, not a problem.  I'll try to get it done this week.